README.md

```diff
@@ -1 +1,6 @@
-# PySATL-CPD-Module
\ No newline at end of file
+# PySATL-CPD-Module
+
+Dependencies installation
+``` shell
+poetry install / poetry install --with dev
```

Comment:
Also we can split this in two lines so it is possible to run in IDE with a green button near the shell script. :)